packages/framework/src/Model/Product/Search/Export/ProductSearchExporter.php

```diff
@@ -128,18 +128,25 @@ public function setEntityManager(EntityManagerInterface $entityManager): void
     /**
      * @param int $domainId
      * @param string $locale
-     * @deprecated Use `exportWithOutput` instead
      */
     public function export(int $domainId, string $locale): void
```

Comment:
Now this looks like a DRY violation. Maybe it could just call the `exportWithOutput` with null IO like this:
```php
$this->exportWithOutput(new SymfonyStyle(new StringInput(''), new NullOutput()));
```

Reply:
Great idea, I just tried
```php
$symfonyStyle = new SymfonyStyle(new ArrayInput([]), new NullOutput());
$this->exportWithOutput($domainId, $locale, $symfonyStyle);
```
and works as expected... My choice is `ArrayInput` as `StringInput` is trying to parse the string and that is unnecessary operation. But in the end it sort of doesn't matter

Reply:
Also we can leave that method `@deprecated` and use this approach directly in the `ProductSearchExportCronModule`, which would be more appropriate in a patch version (considering this will be a part of `v7.3.1`).

Reply:
so will that be deprecated or not? i need to work with that in #1193

Reply:
Hi, sorry for the late reply.
I think it's better to keep the export method (= remove deprecation) so this `emptySymfonyStyle` will remain together and the public interface of a class will be more clear.

I updated the PR, please check it out ;)

Reply:
Edit: after internal discussion updated according to #1193